static/js/subs.js

```diff
@@ -563,11 +570,37 @@ exports.setup_page = function (callback) {
         exports.actually_filter_streams();
         stream_create.set_up_handlers();
 
+        $("#add_new_subscription_extend input[type='text']").on("input", function () {
+            // Debounce filtering in case a user is typing quickly
+            filter_streams();
+        });
+
         $("#add_new_subscription input[type='text']").on("input", function () {
             // Debounce filtering in case a user is typing quickly
             filter_streams();
         });
 
+        $(window).resize(function () {
+            const $search_box_up = $("#add_new_subscription input[type='text']");
```

Comment:
Can you add comments explaining this code?  Why are we modifying search text during a window resize?  That seems weird.

Reply:
I've added comments explaining what's happening.

Just to put it here:
The idea is to keep the two search boxes in sync - by copying the contents of one into the other whenever there is a possibility of them switching - which is on window resize. I figure this will be more efficient than _always_ keeping them in sync, because window resize (while doing a search) is a fairly rare event (as compared to simply doing a search).

The sync between the two boxes as you see in this screen-capture is due to the above code.
![zulip-resize](https://user-images.githubusercontent.com/43912285/77233713-611e4f00-6bcf-11ea-96bc-a6c97a328986.gif)